src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/Details.cshtml

```diff
@@ -29,7 +29,7 @@
                         int ordinal = 0;
                         foreach (var subScheme in Model.Scheme.SubSchemes)
                         {
-                            @await Html.PartialAsync("_AccordionSectionPartial", ("subscheme", ordinal++, subScheme))
+                            @await Html.PartialAsync("_AccordionSectionPartial", ("subscheme", ++ordinal, subScheme))
```

Comment:
Might be best to leave this as is, in case the automation tests use the current title id.

Reply:
Reverted as requested